sdks/java/io/tika/src/main/java/org/apache/beam/sdk/io/tika/TikaOptions.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.tika;
+
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.Validation;
+
+/**
+ * TikaInput Options to support the command-line applications.
+ */
+public interface TikaOptions extends PipelineOptions {
```

Comment:
Just curious: why using a `PipelineOptions` more than a custom interface ? I'm not against, it just a bit weird as it's more IO options than pipeline options actually.

Reply:
Hi JB, thanks, I probably thought it was needed or may be copied it from somewhere, let me try to refactor

Reply:
Hi JB, the idea was to support both TikaIO.fromXXX and fromOptions(TikaOptions), with the latter method offering a quick connection from the main(String args) to TikaIO. So if you are OK then lets keep TikaOptions extending PipelineOptions...

Reply:
Fair enough.